hack/testing/logging.sh

```diff
@@ -33,6 +33,8 @@ USE_LOGGING_DEPLOYER_SCRIPT=
 ENABLE_OPS_CLUSTER=${ENABLE_OPS_CLUSTER:-false}
 DEBUG_FAILURES=${DEBUG_FAILURES:-false}
 USE_LOCAL_SOURCE=${USE_LOCAL_SOURCE:-false}
+ES_VOLUME=${ES_VOLUME:-/var/lib/es}
+ES_OPS_VOLUME=${ES_OPS_VOLUME:-/var/lib/es-ops}
```

Comment:
Yes.  We're just using the local disk on the (virtual) machine.  When the machine goes away, so will the disk.  We could eventually mount a real persistent volume, wipe out the machine, create a new machine with the same pv, and see if es still works.